source/common/http/http1/codec_impl.cc

```diff
@@ -904,6 +904,11 @@ void ServerConnectionImpl::dumpAdditionalState(std::ostream& os, int indent_leve
                                !active_request_.value().request_url_.getStringView().empty()
                            ? active_request_.value().request_url_.getStringView()
                            : "null");
+  os << DUMP_MEMBER_AS(
```

Comment:
Thanks so much for adding this extra state to the dump.

Instead of adding a second bunch of messy code, what do you think of having a dumpState function for the active request, and then just calling DUMP_DETAILS(active_request_)

Reply:
Thanks for the suggestion Alyssa, I will implement this.